### ISSUE TITLE
scss-lint failed with error code: 66

### ISSUE THREAD

**Issue by jthowardesigns** [ahmednuaman/grunt-scss-lint#18]
Every time I run the grunt task for the scss lint i get this error
"scss-lint failed with error code: 66"

**Comment by ahmednuaman** [comment]
Interesting, this may be a path issue, can you try an absolute path instead of relative? Eg for `<%= path.source %>` try setting that as `process.cwd()`.

**Comment by KrisOlszewski** [comment]
I fixed it, but had to rewrite it from scratch. No clue what was causing it. Anyways, great tool and definitely makes my code more consistent!

**Comment by timbergus** [comment]
What I have found using `grunt scsslint --debug --verbose` is that in my case, the problem was that I had no __.scss-lint.yml__ file, so the task didn't find it. I have just change my code like this and it worked just great:

```javascript
scsslint: {
    options: {
        colorizeOutput: true,
        config: null
    },
    src: ['scss/**/*.scss']
}
```

**Comment by danprince** [comment]
I ran into the same issue as @timbergus but without those flags. However, I fixed it in the same way. Maybe worth defaulting the config option to `null` if there's no __`.scss-lint.yml`__ file?

**Comment by jakezatecky** [comment]
Same issue as @timbergus and @danprince. Needed to set `config` to `null`.

**Comment by ajaypatelaj** [comment]
Same issue facing 

>> scss-lint failed with error code: 66
>> and the following message:Error: Command failed:
